From 152f0df19c4dcd0c9bc629b26d166c3af8494eb1 Mon Sep 17 00:00:00 2001
From: brittany <brittany.wald@joyent.com>
Date: Thu, 14 Sep 2017 22:31:08 +0000
Subject: [PATCH] MANTA-3382 node-fash data structure exposure Reviewed by:
 Richard Bradley <richard.bradley@joyent.com> Approved by: Richard Bradley
 <richard.bradley@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 5fd45f4..61ffa4a 100644
--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
         "cmdln": "3.2.1",
         "cmdutil": "1.0.0",
         "extsprintf": "1.3.0",
-        "fash": "2.4.1",
+        "fash": "2.5.0",
         "forkexec": "0.1.0",
         "hogan.js": "2.0.0",
         "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#65bba66818",
-- 
2.21.0

